src/lib.rs

```diff
@@ -33,14 +35,15 @@
 //! above, can be found [here](https://docs.microsoft.com/en-us/uwp/api/Windows.Foundation.Uri?view=winrt-19041). Of
 //! course, when using these APIs from Rust, you will have to remember to translate CamelCase to snake_case as is
 //! the convention in Rust.
-//!
-//! This program will print the following output:
-//!
-//! ```text
-//! domain: kennykerr.ca
-//! port: 80
-//! string: http://kennykerr.ca/
-//! ```
+
+mod bindings {
+    include!(concat!(env!("OUT_DIR"), "/winrt.rs"));
+}
+
+extern crate self as winrt;
```

Comment:
Does this show up in the docs? Perhaps we want to do `#[doc(hidden}]` for this.

Reply:
Done.